language-support/hs/bindings/src/DA/Ledger/Types.hs

```diff
@@ -242,27 +251,17 @@ newtype ApplicationId = ApplicationId { unApplicationId :: Text } deriving (Eq,O
 newtype CommandId = CommandId { unCommandId :: Text } deriving (Eq,Ord,Show)
 newtype PackageId = PackageId { unPackageId :: Text } deriving (Eq,Ord,Show)
 newtype ConstructorId = ConstructorId { unConstructorId :: Text } deriving (Eq,Ord,Show)
-newtype VariantId = VariantId { unVariantId :: Text } deriving (Eq,Ord,Show)
 
 newtype Choice = Choice { unChoice :: Text } deriving (Eq,Ord,Show)
 
 newtype Party = Party { unParty :: Text } deriving (Eq,Ord)
-instance Show Party where show = Text.unpack . unParty -- TODO: really?
+instance Show Party where show = Text.unpack . unParty
```

Comment:
The `show` instance for `Party` in DAML wraps the party name in single quote, eg. `'Alice'`. For the sake of consistency, we should do this here as well.

Reply:
ok